src/types/models/SensorData.ts

```diff
@@ -1,10 +1,33 @@
+// eslint-disable-next-line max-classes-per-file
 export interface SensorData {
   date: Date;
   data: AirImpurityData[];
-  name?: string;
+  code: string;
 }
 
 export interface AirImpurityData {
   name: string;
   value: number;
 }
+
+export class SensorDataClass implements SensorData {
+  code: string;
+  data: AirImpurityData[];
+  date: Date;
+
+  constructor(code: string, data: AirImpurityData[], date: Date) {
+    this.code = code;
+    this.data = data;
+    this.date = date;
+  }
+}
+
+export class AirImpurityDataClass implements AirImpurityData {
+  name: string;
+  value: number;
+
+  constructor(name: string, value: number) {
+    this.name = name;
+    this.value = value;
+  }
+}
```

Comment:
These classes aren't actually types. If they model the real data (not their type but hold the data itself), move them to `src/models` directory. I would also suggest changing the `Class` word to `Model` in the classname as it will be more meaningful.